code/game/machinery/vendors/generic_vendors.dm

```diff
@@ -1,12 +1,30 @@
 //general-use clothing vendors, wardrobe vendors are in another file
 /obj/machinery/economy/vending/assist
-	ads_list = list("Only the finest!",  "Have some tools.",  "The most robust equipment.",  "The finest gear in space!")
-	products = list(/obj/item/assembly/prox_sensor = 4, /obj/item/assembly/igniter = 4, /obj/item/assembly/signaler = 4,
-						/obj/item/wirecutters = 2, /obj/item/cartridge/signal = 4)
-	contraband = list(/obj/item/flashlight = 4, /obj/item/assembly/timer = 2, /obj/item/assembly/voice = 2, /obj/item/assembly/health = 2)
-	prices = list(/obj/item/assembly/prox_sensor = 20, /obj/item/assembly/igniter = 20, /obj/item/assembly/signaler = 30,
-					/obj/item/wirecutters = 50, /obj/item/cartridge/signal = 75, /obj/item/flashlight = 40,
-					/obj/item/assembly/timer = 20, /obj/item/assembly/voice = 20, /obj/item/assembly/health = 20)
+	ads_list = list("Only the finest!",
+					"Have some tools.",
+					"The most robust equipment.",
+					"The finest gear in space!")
+
+	products = list(/obj/item/assembly/prox_sensor = 4,
+					/obj/item/assembly/igniter = 4, /obj/item/assembly/signaler = 4,
+					/obj/item/wirecutters = 2,
```

Comment:
```suggestion
	products = list(/obj/item/assembly/prox_sensor = 4,
					/obj/item/assembly/igniter = 4,
					/obj/item/assembly/signaler = 4,
					/obj/item/wirecutters = 2,
```

Reply:
Surprised i only missed one of these, done.